synapse/events/third_party_rules.py

```diff
@@ -12,9 +12,67 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Optional
 
 from twisted.internet import defer
 
+from synapse.api.constants import EventTypes
+from synapse.events import EventBase
+from synapse.types import JsonDict, create_requester
+
+
+class EventCreator:
+    def __init__(self, hs):
+        self.event_creation_handler = hs.get_event_creation_handler()
```

Comment:
This should be `self._event_creation_handler` to discourage third party modules from reaching in and using it.